cves/CVE-2016-3092.yml

```diff
@@ -10,21 +10,21 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 20
 
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 
 reported_instructions: |
   Was there a date that this vulnerability was reported to the team? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-reported:
+reported: 2016-05-09
```

Comment:
Genuinely curious curious where you got this date. The earliest I found was the publish date from Apache mailing list of 06/23/16 https://mail-archives.apache.org/mod_mbox/www-announce/201606.mbox/%3C45A20804-ABFF-4FED-A297-69AC95AB9A3F@apache.org%3E

Worried I may be overlooking some site from Apache that shows the actual report date to the team.

Reply:
I got it from here: https://tomcat.apache.org/security-9.html#Fixed_in_Apache_Tomcat_9.0.0.M8